APE2.rst

```diff
@@ -198,3 +198,21 @@ original proposal was the addition of the idea that *less than* six month
 releases were acceptable. The above content reflects those suggestions, and the
 APE was accepted 12/10/13, as there were no significant objections from the
 community.
+
+Addenda
+-------
+
+2017 January: In the original text above, feature releases are allowed to make
+backwards incompatible changes, as long as this is mentioned in the changelog.
+In practice, this poses difficulties for packages that rely on astropy 
+(affiliated or otherwise), which generally try to support both the LTS release
+as well as the latest feature release. E.g., the change of the keyword
+``clobber`` to ``overwrite`` in version 1.3 caused code that needed to support
+both 1.0.x and 1.3 a choice of (1) living with giving the users deprecation
+warnings in 1.3 by sticking to ``clobber``; (2) not supporting 1.0.x by moving
+to ``overwrite``; or (3) inserting astropy version checks.
+
+These problems should be avoided by adding the deprecating warnings only in
+LTS versions (n.0.0), and then doing the actual removal at the next feature
+release (n.1.0). Feature releases beyond that should not remove or deprecate
```

Comment:
I agree with @bsipocz here.  Why not allow removals or deprecations in `n.2.0` or `n.3.0` releases?